core/src/repair_service.rs

```diff
@@ -362,12 +377,13 @@ impl RepairService {
             cluster_info.write().unwrap().push_epoch_slots(
                 id,
                 latest_known_root,
+                lowest_slot,
                 slots_in_gossip.clone(),
             );
         }
     }
 
-    fn retain_slots_greater_than_root(slot_set: &mut BTreeSet<u64>, root: u64) {
+    fn retain_slots_greater_than_root(slot_set: &mut BTreeSet<u64>, root: Slot) {
```

Comment:
[nits] Greedy me wishes more `Slot`s instead of `u64` in `BTreeSet<u64>`... Yeah, that helps readability a lot!

Reply:
Oohh this one slipped through. I'll fix it. Thanks!